app/Livewire/Project/Shared/HealthChecks.php

```
@@ -11,26 +11,99 @@ class HealthChecks extends Component
+            $this->healthCheckScheme = $this->resource->health_check_scheme;
+            $this->healthCheckHost = $this->resource->health_check_host;
+            $this->healthCheckPort = $this->resource->health_check_port;
+            $this->healthCheckPath = $this->resource->health_check_path;
+            $this->healthCheckReturnCode = $this->resource->health_check_return_code;
+            $this->healthCheckResponseText = $this->resource->health_check_response_text;
+            $this->healthCheckInterval = $this->resource->health_check_interval;
+            $this->healthCheckTimeout = $this->resource->health_check_timeout;
+            $this->healthCheckRetries = $this->resource->health_check_retries;
+            $this->healthCheckStartPeriod = $this->resource->health_check_start_period;
+            $this->customHealthcheckFound = $this->resource->custom_healthcheck_found;
+        }
```

PHPMD (2.15.0)

The method syncData uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)